src/olympia/scanners/tasks.py

```diff
@@ -14,7 +14,17 @@
 from olympia.amo.celery import create_chunked_tasks_signatures, task
 from olympia.amo.decorators import use_primary_db
 from olympia.constants.scanners import (
-    ABORTED, ABORTING, COMPLETED, CUSTOMS, NEW, RUNNING, SCANNERS, WAT, YARA)
+    ABORTED,
+    ABORTING,
+    COMPLETED,
+    CUSTOMS,
+    ML_API,
+    NEW,
+    RUNNING,
+    SCANNERS,
+    WAT,
+    YARA
```

Comment:
needs a trailing comma if you're going to go this style 😛

Reply:
can't wait to use Black.